geti_sdk/utils/workspace_helpers.py

```diff
@@ -35,7 +35,7 @@ def get_default_workspace_id(rest_session: GetiSession) -> str:
             f"Unexpected response from cluster: {workspaces}. Expected to receive a "
             f"dictionary containing workspace data."
         )
-    default_workspace_names = ["default", "default workspace"]
+    default_workspace_names = ["default", "default workspace", "default_workspace"]
```

Comment:
@ljcornel, we are going to allow the users to change the workspace name soon, so I would recommend not using the names for getting the default one. It is better to ask the user the choose the workspace, or sorting them using he creation time and taking the oldest one.